raiden/messages.py

```diff
@@ -561,8 +568,8 @@ def from_event(cls, event):
 
 
 @dataclass(repr=False, eq=False)
-class WithdrawRequest(SignedRetrieableMessage):
-    """ Requests a signed on-chain withdraw confirmation from partner. """
+class WithdrawBase(SignedRetrieableMessage):
```

Comment:
I see that you're trying to reduce some lines of code here, but is this worth it? I'm not a huge fan of deep hierarchies ...

Reply:
I saw that all 3 descendants of this class have exactly the same attributes so i moved them to a base class. I can flatten them back or we can do it later on if we don't see this structure fitting.